internal/provider/none/installConfig.go

```diff
@@ -5,10 +5,36 @@ import (
 	"github.com/openshift/assisted-service/internal/common"
 	"github.com/openshift/assisted-service/internal/host/hostutil"
 	"github.com/openshift/assisted-service/internal/installcfg"
+	"github.com/openshift/assisted-service/internal/network"
 	"github.com/openshift/assisted-service/internal/provider"
 	"github.com/openshift/assisted-service/models"
+	"github.com/samber/lo"
 )
 
+func (p noneProvider) replaceMachineNetworkIfNeeded(cluster *common.Cluster, cfg *installcfg.InstallerConfigBaremetal) {
+	bootstrapHost := common.GetBootstrapHost(cluster)
+	if bootstrapHost != nil {
```

Comment:
lets change to bootstrap == nil {return}
Should not we return error in this case?

Reply:
OK

Reply:
> lets change to bootstrap == nil {return} 

OK

> Should not we return error in this case?

OK.  Will return error